country_codes.py

```diff
@@ -49,5 +52,4 @@
 if country in countries:
     print(countries[country])
 else:
-    print("Not in country dict.")
-    exit(1)
+    exit(2)
```

Comment:
the different exit codes are so we can handle errors properly in the main script